draft-ietf-rtcweb-jsep.xml

```diff
@@ -2315,7 +2315,42 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
         bounds, or cannot be applied, processing MUST stop and an error MUST be
         returned.
           <list style="symbols">
-            <t>TODO</t>
+            <t>Basic sanity checking is applied for the following attributes in the media section:
+              <list style="symbols">
+                <t>ICE lite: MUST NOT be present.</t>
+                <t>ICE ufrag: MUST be present, and MUST meet the length criteria from [TODO: ref]</t>
+                <t>ICE pwd: MUST be present, and MUST meet the length criteria from [TODO: ref]</t>
+                <t>DTLS fingerprint: MUST be present, and, for existing media sections, MUST be unchanged.</t>
+                <t>DTLS setup: for new media sections, MUST follow the guidance from [TODO: ref]; for existing media sections, MUST be unchanged. [TODO: check whether this is what we agreed on (vs actpass)]</t>
+              </list>
+            </t>
+            <t>If this media section is new, begin gathering candidates for it,
+             as defined in <xref target="RFC5245"/>, Section X.Y.,
+             unless it has been marked as bundle-only.</t>
+            <t>Or, if the ICE ufrag and password values have changed, trigger the ICE Agent
+              to start an ICE restart and begin gathering new candidates for the
+              media section, as defined in <xref target="RFC5245"/>, Section X.Y.,
+              unless it has been marked as bundle-only.</t>
+            <t>If the media section proto value indicates use of RTP:
+              <list style="symbols">
+                <t>If RTCP mux is indicated, prepare to demux RTP and RTCP from the RTP
+                  ICE component. [TODO: ref]</t>
+                <t>For each specified RTP header extension, establish a mapping between
+                  the extension ID and URI. If any RTP header extension is unknown, this
+                  MUST result in an error. [TODO: ref]</t>
+                <t>Prepare to demux RTP data intended for this media section based on
+                  the MID header extension. [TODO: ref]</t>
+                <t>For each specified payload type, establish a mapping between the
+                  payload type ID and the actual media format. [TODO: ref] If any indicated payload
+                  type is unknown, this MUST result in an error.</t>
+                <t>For each specified "rtx" media format, establish a mapping between
+                  the RTX payload type and its associated primary payload type. [TODO: ref] If any
+                  referenced primary payload types are not present, this MUST result in an error.</t>
+                <t>If the directional attribute is of type "sendrecv" or
+                  "recvonly", enable receipt, decoding, and playout of media.
```

Comment:
I think it's more than just decoding; it's the ability to receive any RTP media at all.

I couldn't think of anything better right now, so I just removed the mention of playout.